docs/guide.rst

```diff
@@ -196,31 +196,116 @@ the scene as desired::
 
 .. image:: _static/guide_rotating_cube.gif
 
+
 World objects
 -------------
 
+We've briefly mentioned world objects, materials, and geometry. But how do these relate?
+
 A world object represents an object in the world. It has a transform, by which the
 object can be positioned (translated, rotated, and scaled), and has a visibility property.
 These properties apply to the object itself as well as its children (and their children, etc.).
 
+All objects that have an appearance in the scene are world objects. But there
+are also helper objects, lights, and cameras. These are all world objects.
 
-Geometry
---------
+**Geometry**
 
-Each world object has a geometry. This geometry object contains the
+Most world objects have a geometry. This geometry object contains the
 data that defines (the shape of) the object, such as positions, plus
 data associated with these positions (normals, texcoords, colors, etc.).
 Multiple world objects may share a geometry.
 
 
-Materials
----------
+**Materials**
 
-Each world object also has a material. This material object defines the
+All visible world objects have a material. This material object defines the
```

Comment:
I would remove "visible", invisible objects can have materials too right?

Reply:
I meant visible as in having an apperance, so excluding lights, cameras, groups, etc. I think I'll change it to say "Most".

Reply:
I understand, I just worry that our readers may take it literally :)